backend-opal/src/server/db/config.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+require('dotenv').config({ path: path.join(__dirname, '/./../../../.env') });
+const mysql = require('mysql');
+
+const DB_CONFIG = {
```

Comment:
@Yuan-EPAM why the capital naming? its a plain configuration object, can be `dbConfig` or just `config`